packages/angular/src/app/pages/crm-contact-details/crm-contact-details.component.scss

```diff
@@ -28,11 +35,13 @@ $left-panel-width: 400px;
     gap: 20px;
 
     .left {
-      flex: 1 400px;
+      flex: 1 $left-panel-width;
+      margin-top: 8px;
```

Comment:
I see that we use this value in many places. Looks like we can create variable for this

Reply:
in next pr we remake this part totally